docs/source/commands/supergraphs.mdx

```diff
@@ -21,7 +21,7 @@ Rover commands that interact with supergraphs begin with `rover supergraph`. The
 
 ## Fetching a supergraph schema from GraphOS
 
-### `supergraph fetch`
+#### `supergraph fetch`
```

Comment:
Using H4s for this so they don't appear in the table of contents (they are essentially the raw rover command version of the H2 above)